hvpy/io.py

```diff
@@ -54,4 +53,19 @@ def url(self) -> str:
         """
         Final API endpoint URL.
         """
-        return BASE_URL + self.__class__.__name__[:-15] + "/"
+        base_url = get_api_url()
+        return base_url + self.__class__.__name__[:-15] + "/"
```

Comment:
```suggestion
        os.environ.clear()
        return base_url + self.__class__.__name__[:-15] + "/"
```
Tested this locally, and it passes all the tests. 
Should we do it?

Reply:
I don't think we should modifying a user's environment when someone uses hvpy.